src/server/capacity_unit_calculator.cpp

```diff
@@ -40,6 +40,44 @@ capacity_unit_calculator::capacity_unit_calculator(replica_base *r) : replica_ba
                                           name,
                                           COUNTER_TYPE_VOLATILE_NUMBER,
                                           "statistic the recent write capacity units");
+
+    snprintf(name, 255, "recent_get_throughput@%s", str_gpid.c_str());
```

Comment:
Rename `recent_get_throughput` to `get_bytes`. Because the prefixed "recent_" is redundant and verbose. And "throughput" is not clear enough to tell what the unit is.

Reply:
if delete `recent_`, the counter type `COUNTER_TYPE_VOLATILE_NUMBER` should be changed into `COUNTER_TYPE_RATE`?

Reply:
Yes. Similar with QPS counters.